scripts/utils/termux/alternatives/termux_alternatives.sh

```diff
@@ -0,0 +1,56 @@
+# shellcheck shell=bash
+
+# Title:          termux_alternatives
+# Description:    A library for working with Termux package Alternatives.
+
+
+
+##
+# Parse datas from the alternatives file and put the datas into the variables.
+# .
+# .
+# **Parameters:**
+# `path_to_alternatives_file` - The path to the package's `.alternatives` file.
+# .
+# **Returns:**
+# returns their contents in the associative arrays:
+# ${LINK[@]} ${[ALTERNATIVE[@]} ${DEPENDENTS[@]} ${PRIORITY[@]}
+# .
+# .
+# termux_alternatives__parse_alternatives_file <path_to_alternatives_file>
+##
```

Comment:
I think the rewritten function comment is overly verbose and loses some of the original information.

I'm also not sure why this got moved to a new function (pseudo-)namespace.
`termux_parse_alternatives` or `termux_parse_alternatives_file` should be plenty descriptive.

Reply:
I needed to move the function to utils so that it could be used not only in `termux_step_update_alternatives` but also in `termux_step_install_switcher_files`.

Regarding the function naming and its comments/description, this is a standard for scripts in utils (at least in `scripts/utils/termux/`). You can ask @agnostic-apollo for more details. Also, take a look at another script with the same standard - https://github.com/termux/termux-packages/blob/master/scripts/utils/termux/package/termux_package.sh